Description: Fix armel FTBFS
 Do not include <GL/gl.h>
 explicitly, it conflicts with <GLES2/gl2.h> used on armel.
Author: Jani Monoses <jani@ubuntu.com>
Bug-Debian: http://bugs.debian.org/649167
Index: clutter-gesture-0.0.2.1/clutter-gesture/clutter-gesture.c
===================================================================
--- clutter-gesture-0.0.2.1.orig/clutter-gesture/clutter-gesture.c	2010-01-06 12:11:44.000000000 +0800
+++ clutter-gesture-0.0.2.1/clutter-gesture/clutter-gesture.c	2011-11-20 15:04:21.270750419 +0800
@@ -29,7 +29,6 @@
 #include <string.h>
 #include <cairo/cairo.h>
 #include <pango/pangocairo.h>
-#include <GL/gl.h>
 #include <glib-object.h>
 
 #include "clutter-gesture.h"
Index: clutter-gesture-0.0.2.1/engine/engine.c
===================================================================
--- clutter-gesture-0.0.2.1.orig/engine/engine.c	2010-01-06 12:11:44.000000000 +0800
+++ clutter-gesture-0.0.2.1/engine/engine.c	2011-11-20 15:04:31.242596102 +0800
@@ -26,7 +26,6 @@
 #include <errno.h>
 #include <cairo/cairo.h>
 #include <pango/pangocairo.h>
-#include <GL/gl.h>
 #include "engine.h"
 #include "../clutter-gesture/clutter-gesture.h"
 #include "stroke.h"
